ui/src/app/applications/components/resource-icon.tsx

```diff
@@ -14,7 +14,7 @@ export const ResourceIcon = ({kind}: {kind: string}) => {
     const style: React.CSSProperties = {
         display: 'inline-block',
         verticalAlign: 'middle',
-        padding: '2px 4px',
+        padding: `${n <= 2 ? 2 : 0}px 4px`,
```

Comment:
I think it should be `2px ${n <= 2 ? 2 : 0}px`, is not it? Hope I'm not missing anything.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/padding

```css
padding: 10% 0;
```
![image](https://user-images.githubusercontent.com/2119212/91591738-5fe61080-e92b-11ea-9cfd-9ba7d4f428f9.png)

I'm trying to change top-bottom alignment, which is the first unit.

Reply:
Ah, got it. I was looking at the wrong element. Thank you!